tests/end_to_end/repo/.github/workflows/ci.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         uses: py-cov-action/python-coverage-comment-action@__ACTION_REF__
         with:
           GITHUB_TOKEN: ${{ github.token }}
+          ANNOTATE_MISSING_LINES: true
```

Comment:
nice ! do you think we should check that they were created properly or is it just intended for humans to have a look ?

Reply:
It was mainly meant for me to actually see the feature in action in a repo, so I know if everything works the way I envision it.
I don't think we necessarily need to verify that the annotations work (I don't know how easy that would be either), but if you already have a concrete idea how to do that, we could include it.